justfile

```diff
@@ -29,3 +30,14 @@ down:
 # Pull all docker images
 pull:
     {{ DC }} pull
+
+@test-int test_path="":
+    echo "Starting up spd-lookup services for integration testing"
+    {{ DC }} up -d &> /dev/null
+    ( while [ "$(docker logs -n 1 $(docker ps | grep spd_lookup_api | awk -F ' ' '{print $1}'))" != "starting server on port 1312" ] ; do sleep 1s; done ) &> /dev/null

```

Comment:
Same comment here as SeattleDSA/signal-scanner-bot#83 - this can be `{{ DC }} logs -n 1` I believe

Reply:
This is a good point. Will update to include.